libs/utils/test.py

```diff
@@ -42,20 +42,31 @@ def _init(cls, conf, *args, **kwargs):
             cls.logger.setLevel(kwargs['loglevel'])
             kwargs.pop('loglevel')
 
-        cls.conf = conf
-
         cls._runExperiments()
 
     @classmethod
+    def _getTestConf(cls):
+        if not hasattr(cls, "test_conf"):
```

Comment:
Agreed, that would be clearer. I don't think we should put a sanity check in this method since subclasses might override this method without calling `super..._getTestConf`.